doc/src/docs/InputOutputReference/01b-InputOutputReference.md

```diff
@@ -17398,7 +17386,7 @@ NOTE: branch object required only when tank use inlet nodes are used.
     Central HPWH Branch,     !- Name
     0,                       !- Maximum Flow Rate {m3/s}
     ,                        !- Pressure Drop Curve Name
-    WaterHeater:HeatPump,    !- Component 1 Object Type
```

Comment:
Some github weirdness is going on here. It's just in the diff display. It looks like it's skipping from 17372 to 17386 in the diff view without indicating that the skip is happening. When I open up the file that is committed here's what's there:

<hr/>

```idf
WaterHeater:HeatPump:PumpedCondenser,
    PlantHeatPumpWaterHeater,!- Name
    PlantHPWHSch,            !- Availability Schedule Name
    PlantHPWHTempSch,        !- Compressor Setpoint Temperature Schedule Name
    2.0,                     !- Dead Band Temperature Difference {deltaC}
    HPPlantWaterInletNode,   !- Condenser Water Inlet Node Name
    HPPlantWaterOutletNode,  !- Condenser Water Outlet Node Name
    0.00115525,              !- Condenser Water Flow Rate {m3/s}
    1.00695,                 !- Evaporator Air Flow Rate {m3/s}
    OutdoorAirOnly,          !- Inlet Air Configuration
    ,                        !- Air Inlet Node Name
    ,                        !- Air Outlet Node Name
    HPPlantAirInletNode,     !- Outdoor Air Node Name
    HPPlantAirOutletNode,    !- Exhaust Air Node Name
    ,                        !- Inlet Air Temperature Schedule Name
    ,                        !- Inlet Air Humidity Schedule Name
    ,                        !- Inlet Air Zone Name
    HPWHPlantTank,           !- Tank Name
    HPWH Use Inlet Node,     !- Tank Use Side Inlet Node Name
    HPWH Use Outlet Node,    !- Tank Use Side Outlet Node Name
    HPWHPlantDXCoil,         !- DX Coil Name
    11.0,                    !- Minimum Inlet Air Temperature for Compressor Operation {C}
    Outdoors,                !- Compressor Location
    ,                        !- Compressor Ambient Temperature Schedule Name
    HPWHPlantFan,            !- Fan Name
    BlowThrough,             !- Fan Placement
    ,                        !- On Cycle Parasitic Electric Load {W}
    ,                        !- Off Cycle Parasitic Electric Load {W}
    ;                        !- Parasitic Heat Rejection Location
```




NOTE: branch object required only when tank use inlet nodes are used.



```idf
  Branch,
    Central HPWH Branch,     !- Name
    0,                       !- Maximum Flow Rate {m3/s}
    ,                        !- Pressure Drop Curve Name
    WaterHeater:HeatPump:PumpedCondenser,    !- Component 1 Object Type
    PlantHeatPumpWaterHeater,!- Component 1 Name
    HPWH Use Inlet Node,     !- Component 1 Inlet Node Name
    HPWH Use Outlet Node,    !- Component 1 Outlet Node Name
    PASSIVE;                 !- Component 1 Branch Control Type




  WaterHeater:Mixed,
    HPWHPlantTank,           !- Name
    0.757,                   !- Tank Volume {m3}
    Plant Hot Water Setpoint Temp Schedule,  !- Setpoint Temperature Schedule Name
    2.0,                     !- Deadband Temperature Difference {deltaC}
    82.2222,                 !- Maximum Temperature Limit {C}
    CYCLE,                   !- Heater Control Type
    25000,                   !- Heater Maximum Capacity {W}
    0,                       !- Heater Minimum Capacity {W}
    ,                        !- Heater Ignition Minimum Flow Rate {m3/s}
    ,                        !- Heater Ignition Delay {s}
    ELECTRICITY,             !- Heater Fuel Type
    0.98,                    !- Heater Thermal Efficiency
    ,                        !- Part Load Factor Curve Name
    10,                      !- Off Cycle Parasitic Fuel Consumption Rate {W}
    ELECTRICITY,             !- Off Cycle Parasitic Fuel Type
    0,                       !- Off Cycle Parasitic Heat Fraction to Tank
    30,                      !- On Cycle Parasitic Fuel Consumption Rate {W}
    ELECTRICITY,             !- On Cycle Parasitic Fuel Type
    0,                       !- On Cycle Parasitic Heat Fraction to Tank
    Outdoors,                !- Ambient Temperature Indicator
    ,                        !- Ambient Temperature Schedule Name
    ,                        !- Ambient Temperature Zone Name
    HPWHPlantTank OA Node,   !- Ambient Temperature Outdoor Air Node Name
    0.0,                     !- Off Cycle Loss Coefficient to Ambient Temperature {W/K}
    0.0,                     !- Off Cycle Loss Fraction to Zone
    0.0,                     !- On Cycle Loss Coefficient to Ambient Temperature {W/K}
    0.0,                     !- On Cycle Loss Fraction to Zone
    ,                        !- Peak Use Flow Rate {m3/s}
    ,                        !- Use Flow Rate Fraction Schedule Name
    ,                        !- Cold Water Supply Temperature Schedule Name
    HPWH Use Inlet Node,     !- Use Side Inlet Node Name
    HPWH Use Outlet Node,    !- Use Side Outlet Node Name
    0.98,                    !- Use Side Effectiveness
    HPPlantWaterOutletNode,  !- Source Side Inlet Node Name
    HPPlantWaterInletNode,   !- Source Side Outlet Node Name
    0.98,                    !- Source Side Effectiveness
    autosize;                !- Use Side Design Flow Rate {m3/s}